doc/help/elektra-spec.md

```diff
@@ -21,7 +21,7 @@ key database semantics (new plugins might be necessary, though).
 The most simple use is to enlist all keys that will be used by an application
 and maybe give a description for them (we use ini syntax in this document):
 
-```
+```ini
```

Comment:
Thanks! But I cannot see on the site what to write in the fence (e.g. for Apache or C++)?

What happens if a unknown language is used? Does it simply ignore the string then?